plugins/tiddlywiki/codemirror/config.tid

```diff
@@ -7,6 +7,7 @@ type: application/json
   ],
   "configuration": {
       "showCursorWhenSelecting": true,
-      "inputStyle": "textarea"
+      "inputStyle": "textarea",
+      "lineNumbers": false
   }
```

Comment:
I think this should be lineNumbers: true

Otherwise the basic editor seems to be the same as core without the plugin.

Reply:
it's enough to just omit that, linenumbers are default
I wanted to add that so that the default is really simple and it doesn't show a linenumber in $:/ControlPanel input fields

I don't care much about it, just thought it makes the basic configuration more basic